charts/kubewarden-controller/chart-values.yaml

```diff
@@ -5,50 +5,44 @@
 nameOverride: ""
 # fullnameOverride completely replaces the generated release name
 fullnameOverride: ""
-
 # Secrets to pull container images from private registries
 imagePullSecrets: []
-
 # -- Additional labels to add to all resources
 additionalLabels: {}
-  # app: kubewarden-controller
+# app: kubewarden-controller
```

Comment:
```suggestion
#   app: kubewarden-controller
```
Since update-cli takes the newlines and empty spaces between commits, this would be incorrectly indented, so adding indentation back.

Reply:
@jvanz / @olblak  is there a way to prevent that from happening again?

Reply:
I suppose after this one fix, the comment is at the starting line and there would be no need for more fixes.